### PR TITLE
Implement Recent Chats Screen

### LINKED ISSUE #7: Implement Recent Chats Screen
**Description:**
Implement recent chats screen. Use the **Users Screen** design from the [designs ](https://github.com/ChatAir/ChatAir-android/tree/master/designs)folder as a reference.

### PR DESCRIPTION
**Pull request by pfieffer**
## Description
- Description of issue:
Implement recent chats screen. Use the Users Screen design from the designs folder as a reference.
- Output before solved:
No Recent Chats Screen
- Changes in Issue:
Add Recent Chats Screen with dummy data and the functionalities needed for the screen
- Output of solved issue:
Splash Screen navigates to the Recent Chats Screen which shows up a list of dummy recent chats. The list item has click listeners attached. The toolbar overflow menu has three dummy options.

## Type of Changes?
Just put an x in the [ ] which are valid.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing Suites

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes.
- [x] ./gradlew assembleDebug assembleRelease

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

closes #7

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,8 +1,14 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
+    <AndroidXmlCodeStyleSettings>
+      <option name="ARRANGEMENT_SETTINGS_MIGRATED_TO_191" value="true" />
+    </AndroidXmlCodeStyleSettings>
     <JetCodeStyleSettings>
       <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
     </JetCodeStyleSettings>
+    <MarkdownNavigatorCodeStyleSettings>
```
Comment (pfieffer) [inline .idea/codeStyles/Project.xml]: I hope this is not a problem, but let me know if this does not work for you guys.

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+      <option name="RIGHT_MARGIN" value="72" />
+    </MarkdownNavigatorCodeStyleSettings>
```
Comment (pfieffer) [inline .idea/codeStyles/Project.xml]: I hope this is not a problem, but let me know if this does not work for you guys.

**Comment by pfieffer** [comment]
I have marked this PR as a **WIP**, meaning there are work to be done.
Work that is pending:

1. FAB icon
2. FAB Click Listener

**Comment by iadeelzafar** [comment]
@pfieffer If you could add a screenshot of the UI in the PR description, that'd be great.

**Comment by pfieffer** [comment]
| Recent Chats Screen | Overflow menu Options | Click on a list item |
| -----------------------------| -------------------------------- | ------------------------ |
| Screenshot_20200115-195852      | Screenshot_20200115-195858 | Screenshot_20200115-195912 |

File: .idea/codeStyles/Project.xml (hunk 2 above)
Comment (iadeelzafar) [reply .idea/codeStyles/Project.xml]: @pfieffer Why the right margin here is 72? I think we can use nodes and connect them in the constraint layout.

Comment (pfieffer) [reply .idea/codeStyles/Project.xml]: This is not actually related to `ConstraintLayout` or `Views` in anyway. As you can see this is in the `.idea/codeStyles/Project.xml ` file, which is a style file for the code. This is specifically for when the IDE encounters markdown files. This pieced of code style applies a right margin value of 72 characters for Markdown files.

Comment (iadeelzafar) [reply .idea/codeStyles/Project.xml]: Gotcha! Thanks for the clarification.

**Comment by iadeelzafar** [comment]
🎉 Thanks for the PR @pfieffer

**Pull request merged**